subsys/bluetooth/controller/ll_sw/lll_sync.h

```diff
@@ -32,6 +32,14 @@ struct lll_sync {
 	uint8_t sca:3;
 #endif /* CONFIG_BT_CTLR_SYNC_ISO */
 
+	/* Counter used by LLL abort of event when in unreserved time space to
+	 * provide near fair scheduling of overlapping multiple Periodic
+	 * Sync sets.
+	 */
+#if defined(CONFIG_BT_CTLR_SCAN_AUX_SYNC_RESERVE_MIN)
```

Comment:
```suggestion
#if defined(CONFIG_BT_CTLR_SCAN_AUX_SYNC_RESERVE_MIN)
	/* Counter used by LLL abort of event when in unreserved time space to
	 * provide near fair scheduling of overlapping multiple Periodic
	 * Sync sets.
	 */
```

Kinda makes more sense to have the comment next to the field

Reply:
done